### PR TITLE
Fix 65 security buttons

### PR DESCRIPTION
**Pull request by alessandrothedev**

### CODE REVIEW

File: src/components/SecurityBar/AlertButton/AlertButton.scss [hunk 1]
```diff
@@ -1,42 +1,37 @@
...
   background-color: $text-tertiary;
   box-shadow: 0px 3.5px 5.5px rgba(0, 0, 0, 0.05);
 
+
   &__icon {
-    
     height: 38px;
     width: 38px;
-    aspect-ratio: 1;
+    // aspect-ratio: 1;
```
Comment (gabagardi) [inline src/components/SecurityBar/AlertButton/AlertButton.scss:19]: remove comment (looks like a comment)

File: src/components/SecurityScore/SecurityScore.scss [hunk 2]
```diff
@@ -10,15 +10,13 @@
   padding: 20px;
   margin-top: 150px;
   margin-left: 50px;
-  // overflow: hidden;
```
Comment (gabagardi) [inline src/components/SecurityScore/SecurityScore.scss:13]: comment

Comment (iagardi) [inline src/components/SecurityScore/SecurityScore.scss:13]: "There is no commented out code and “console.log”-s are removed."

File: src/components/SecurityBar/AlertButton/AlertButton.jsx [hunk 3]
```diff
@@ -4,19 +4,21 @@ import "./AlertButton.scss";
...
   return (
-    <button className="button">
-      <div className={buttonStyle}>
-        <img className="button__icon--svg" src={iconImage} alt="alerts" />
-      </div>
-      <div className="button__info">
-        <p className="button__info--heading">{buttonText}</p>
-        <p className="button__info--number">{buttonNumber}</p>
+    <button className="alert-button">
+      <div className="alert-button-icon-container">
```
Comment (iagardi) [inline src/components/SecurityBar/AlertButton/AlertButton.jsx:13]: Slightly confusing class naming

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**